core/net/rpl/rpl-ext-header.c

```diff
@@ -507,10 +507,10 @@ update_hbh_header(void)
       if((UIP_EXT_HDR_OPT_RPL_BUF->flags & RPL_HDR_OPT_DOWN)) {
         if(uip_ds6_route_lookup(&UIP_IP_BUF->destipaddr) == NULL) {
           UIP_EXT_HDR_OPT_RPL_BUF->flags |= RPL_HDR_OPT_FWD_ERR;
-          PRINTF("RPL forwarding error\n");
+          PRINTF("RPL: RPL forwarding error\n");
```

Comment:
The repetition in "RPL: RPL" can be removed here and in the modified lines below. The rest of the changes look good.